openbas-dev/Backend start.run.xml

```diff
@@ -2,7 +2,6 @@
   <configuration default="false" name="Backend start" type="SpringBootApplicationConfigurationType" factoryName="Spring Boot">
     <option name="ACTIVE_PROFILES" value="dev" />
     <option name="ALTERNATIVE_JRE_PATH" value="$PROJECT_DIR$/../../../../../../../../../Program Files/Java/jdk-22" />
```

Comment:
We probably should delete (or comment) also this line as it's very specific to our workstations

Reply:
Oh sorry, it should ne commited